app/controllers/your-psp/get.controller.test.js

```diff
@@ -36,10 +38,21 @@ describe('Your psp GET controller', () => {
 
 
   it('should pass the enableStripeOnboardingTaskList to the page ', async () => {
-    process.env.ENABLE_STRIPE_ONBOARDING_TASK_LIST = true
 
     await getController(req, res, next)
-   
-    sinon.assert.calledWith(res.render, `your-psp/index`)    
+    
+    const pageData = res.render.args[0][1]
+    expect(pageData.enableStripeOnboardingTaskList).to.equal(false)
+    
+    sinon.assert.calledWith(res.render, 'your-psp/index')
   })
+
+  it('when the enableStripeOnboardingTaskList has not been set ', async () => {
```

Comment:
Update this line to:
`when the enableStripeOnboardingTaskList has not been set then it should not be passed to the page`